week6.ipynb

```diff
@@ -0,0 +1,385 @@
+    "# we started without any pennies suspended from our spring, so our range will start at 0\n",
+    "# the greatest number of pennies that we tested was 20\n",
+    "# and we tested by increments of 5\n",
+    "penny_seq <- seq(from = 0, to = 20, by =5)\n",
+    "\n",
+    "# now we want to convert the number of pennies tested in each trial into a weight\n",
+    "# we can do this by performing an operation on the penny_seq, and saving that result to a new sequence\n",
+    "weight_seq <- penny_seq * weight_of_penny\n",
+    "\n",
+    "# let's perform another operation to make sure the weights are in kg m/s^2 (currently they are g m/s^2)\n",
+    "# we can do this by dividing by 1000\n",
+    "weight_seq_kg <- weight_seq/1000\n",
+    "\n",
+    "# print all arrays to make sure you understand what's happened:\n",
+    "print(cat('penny sequence:', penny_seq))\n",
+    "print(cat('weight sequence:', weight_seq))\n",
+    "print(cat('weight sequence in kg:', weight_seq_kg))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[1] 4.000 4.125 4.250 4.375 4.500\n",
+      "[1] 0.000 0.125 0.250 0.375 0.500\n",
+      "[1] 0.000000000 0.003172589 0.006345178 0.009517766 0.012690355\n"
+     ]
+    }
+   ],
+   "source": [
+    "# now we'll create a vector to store the length and displacement measurements of our spring\n",
+    "# we'll enter these values by hand, in case they're unevenly spaced\n",
+    "# instead of starting at 0, our vector start at 4, because the original length of our rubber band was four inches\n",
+    "length_inches <- c(4, 4.125, 4.25, 4.375, 4.5)\n",
+    "\n",
+    "# however, what we're really interested in is displacement, not length.\n",
+    "# to get displacement, we'll subtract the initial length from the entire vector\n",
+    "displacement_inches <- length_inches - 4\n",
+    "\n",
+    "# we want units in meters, so we'll perform another operation\n",
+    "displacement_m <- displacement_inches/39.4 # there are 39.4 inches in a meter\n",
+    "\n",
+    "# again, print your arrays to make sure you've followed along\n",
+    "print(length_inches)\n",
+    "print(displacement_inches)\n",
+    "print(displacement_m)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "weight sequence kg type:  doubleNULL\n",
+      "weight sequence kg length:  5NULL\n",
+      "displacement m type:  doubleNULL\n",
+      "displacement m length:  5NULL\n"
+     ]
+    }
+   ],
+   "source": [
+    "# when we plot our data, we'll want to make sure that our x and y data are of the same length and type\n",
+    "# check them out now before plotting\n",
```

Comment:
Your comments are extensive (this is great, your code is **very** understandable)! For readability, maybe put them in markdown instead of inline # comments? Just because inline comments don't get line breaks – they don't fit on my screen so I have to scroll to read them.